frontend/src/components/Items.vue

```diff
@@ -127,13 +142,43 @@ export default {
       },
       immediate: true,
     },
+    items: {
+      handler: function () {
+        if (
+          this.items.some((item) => item.hasOwnProperty("activeState")) &&
+          this.mode === 'single'
```

Comment:
1. `this.mode === 'single'` condition should be tested first as
it much faster than iterating over the items with `some`

2. `some((item) => item.hasOwnProperty("activeState")` seems redundant as we check  for `item.ActivateState` inside the for loop. so why traverse the **collection** some((item) => item.hasOwnProperty("activeState") twice?